Packs/SekoiaXDR/Scripts/SekoiaXDRChangeStatus/SekoiaXDRChangeStatus.py

```diff
@@ -0,0 +1,49 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main():
+    incident = demisto.incidents()[0]  # type: ignore
+    isMirrorEnable = incident.get("dbotMirrorDirection")
+    alert_short_id = demisto.args().get("short_id")
+    new_status = demisto.args().get("status")
+    comment = demisto.args().get("comment")
+
+    if new_status in ["Ongoing", "Acknowledged"]:
+        if comment:
+            user = execute_command("getUsers", {"current": "true"})[0]["name"]  # type: ignore
+            execute_command(
+                "sekoia-xdr-post-comment-alert",
+                {"id": alert_short_id, "comment": comment, "author": user},
+            )
+        if isMirrorEnable in ["Out", "Both"]:
+            execute_command("setIncident", {"sekoiaalertstatus": new_status})
+        elif isMirrorEnable == "In":
+            execute_command(
+                "sekoia-xdr-update-status-alert",
+                {"id": alert_short_id, "status": new_status},
+            )
+        else:
+            execute_command(
+                "sekoia-xdr-update-status-alert",
+                {"id": alert_short_id, "status": new_status},
+            )
+            execute_command("setIncident", {"sekoiaalertstatus": new_status})
```

Comment:
Yes of course, Normally if we press the button to update the status. There are 3 cases :
1 / if the Mirroring is out or both so we need just to use `setIncident ` command to change it and it will change in the sekoia side.
2/ If there's just the mirroring In : So we need to update the command in the sekoia side and then automatically it will change in cortex side.
3/ finally if we didn't use mirroring so we need to use both commands.

Reply:
Where this script is being used?
Because if you run this script in XSOAR, and there is only mirroring in or no mirroring at all, then the customer doesn't want to update sekoia with his changes in XSOAR, so cases 2+3 are incorrect.

Reply:
It's working on update status button on the layout. 
Yes you are right, I'm agree with you. But I don't know if we will have some sync data problems ?

Reply:
Yes, but it is up to the customer to decide whether they want to update sekoia as well or not, if they want to update sekoia with the changes in XSOAR, they need to choose mirroring out or both in the configuration of the instance, we shouldn't force the sync.

Reply:
Ok i will change it, thanks for this catch.